extensions/pyRevitCore.extension/pyRevit.tab/pyRevit.panel/Settings.smartbutton/script.py

```diff
@@ -389,42 +388,27 @@ def _setup_addinfiles(self):
             {str(x.Product.ProductYear):x
              for x in PyRevit.PyRevitAttachments.GetAttachments()}
 
-        for rvt_ver, checkbox in self._addinfiles_cboxes.items():
-            if rvt_ver in attachments:
-                if rvt_ver != HOST_APP.version:
-                    checkbox.Content = \
-                        self._make_product_name(
-                            attachments[rvt_ver].Product,
-                            ''
-                            )
+        for rvt_cb in self.supported_revit_versions_CB:
```

Comment:
I would have kept the `checkbox` variable name and added a line with `rvt_ver = checkbox.Version` to minimize the changes and avoid repeated `revit_cb.Version` in the following lines.